rest-server/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(cadg-rest-server)
+
+find_package(Boost REQUIRED COMPONENTS system thread)
+find_package(Threads REQUIRED)
+find_package(OpenSSL 1.0.1 REQUIRED)
+set(OPENSSL_VERSION "1.0.1")
+
+message("-- configuring gcc options")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++17 -DBOOST_LOG_DYN_LINK")
+
+# source files
+add_executable(${PROJECT_NAME}  
+                                ./src/controller/user_controller.cpp
```

Comment:
These are out of order. All the controller directory files should be next to each other, everything should be in alphabetical order.